assets/src/blocks/TopicLink/TopicLinkBlock.js

```diff
@@ -0,0 +1,59 @@
+import {TopicLinkEditor} from './TopicLinkEditor.js';
+
+const {registerBlockType, getBlockTypes} = wp.blocks;
+const {__} = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/topic-link';
+
+export const registerTopicLinkBlock = () => {
+  const blockAlreadyExists = getBlockTypes().find(block => block.name === BLOCK_NAME);
```

Comment:
I don't think this is needed, we added it in some blocks when they are still in the plugin too

Reply:
Fixed!